### PR TITLE
pytest added for tasks

### PR DESCRIPTION
**Pull request by abhin2002**

### CODE REVIEW

File: task_5/test.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+def test_download_multiple_images(downloader):
+    paths = downloader[0:3]  # Download first 3 images
+    assert len(paths) == 3
+    for path in paths:
+        if isinstance(path, str) and not path.startswith('An error occurred'):  # Check if path is a string and not an error message
+            assert os.path.exists(path)
+
+
+if __name__ == "__main__":
+    pytest.main([__file__])
```
Comment (FayazRahman) [inline task_5/test.py]: Is this necessary?

File: task_6/test.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+def test_download_multiple_images(downloader):
+    paths = downloader[0:3]  # Download first 3 images
+    assert len(paths) == 3
+    for path in paths:
+        if isinstance(path, str) and not path.startswith('An error occurred'):  # Check if path is a string and not an error message
+            assert os.path.exists(path)
+
+
+if __name__ == "__main__":
+    pytest.main([__file__])
```
Comment (FayazRahman) [inline task_6/test.py]: Same here

**Review by FayazRahman — COMMENTED** [review]
Remove the `bin` folder

**Review by FayazRahman — CHANGES_REQUESTED** [review]
Remove `bin` folder

File: task_5/test.py (hunk 1 above)
Comment (abhinpt0) [reply task_5/test.py]: No, removed

**Review by FayazRahman — APPROVED** [review]

**Pull request merged**